src/notebook_index.ipynb

```diff
@@ -107,23 +107,24 @@
     "\n",
     "#### Operations\n",
     "Detailed examples on importing and implementing IDAES unit models in flowsheets\n",
-    "  * [compressor](Examples/UnitModels/Operations/compressor.ipynb) - Compressor Unit Model with Span-Wagner Property Package for supercritical CO2\n",
-    "  * [heater](Examples/UnitModels/Operations/heater.ipynb) - Heat a liquid mixture of benzene-toluene using a simple heater unit model and an ideal property package\n",
+    "  * [compressor](Examples/UnitModels/Operations/compressor_testing.ipynb) - Compressor Unit Model with Span-Wagner Property Package for supercritical CO2\n",
+    "  * [feed](Examples/UnitModels/Operations/feed_testing.ipynb) - Feed unit model with ideal property package  \n",
+    "  * [heater](Examples/UnitModels/Operations/heater_testing.ipynb) - Heat a liquid mixture of benzene-toluene using a simple heater unit model and an ideal property package\n",
     "\n",
-    "  * [mixer](Examples/UnitModels/Operations/mixer.ipynb) - Mixer unit model with ideal property package\n",
-    "  * [pump](Examples/UnitModels/Operations/pump.ipynb) - Pump unit model with iapws property package\n",
-    "  * [heat exchanger 0D](Examples/UnitModels/Operations/heat%20exchanger%200D.ipynb) - Heat Exchanger 0D unit model heating a benzene-toluene mixture using steam\n",
-    "  * [skeleton unit](Examples/UnitModels/Operations/skeleton%20unit.ipynb) - Skeleton unit model demonstrating custom variables, ports, and constraints\n",
+    "  * [mixer](Examples/UnitModels/Operations/mixer_testing.ipynb) - Mixer unit model with ideal property package\n",
```

Comment:
Just to confirm, are these changes expected?

Reply:
Yes because the hyperlinks were not working for some of the existing models because they were missing the "_testing" at the end of the name.

Reply:
We should look into this before merging. I'm not 100% familiar with the implementation, but I think the `_testing` prefix is stripped by `build.py` during the build process.

My concern is that if the `_testing` prefix is added to the URL, the links will work pre-build (which I assume is why @MarcusHolly had to do that), but not post-build (which is what ultimately is available online). In other words, the links only work pre-build or post-build, but not both.

If this is the case, this is a limitation of the (soon-to-be-retired) `build.py` infrastructure, and I think we should address this (or at least document it) to make it less confusing for example developers.